doc/applications.md

```diff
@@ -0,0 +1,151 @@
+# Creating applications
```

Comment:
Cool, thanks for taking time to look through it all :) I will read up on esdoc and configure it!